.codecov.yml

```diff
@@ -0,0 +1,2 @@
+codecov:
+  branch: master
```

Comment:
I'll need to double check but I think we can get away without this.  Originally we needed it to set the default branch to develop, but I think without this file, it defaults to master regardless